bitmap-ittybitty/lib/transformer.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+function transform(palette) {
+  for (let i = 0; i < palette.length ; i++) {
+    palette[i] = palette[i] + 2;
+    if (palette[i] > 255) {
+      palette[i] = 255;
+    }
+  }
+  return palette;
+}
+exports.transformAdd2 = transform;
```

Comment:
It's a little weird to name your function one thing, then store it as something else. Inconsistency hurts readability.